identity-apps-core/apps/authentication-portal/src/main/webapp/hyprlogin.jsp

```diff
@@ -164,9 +164,9 @@
                     pollAuthStatus();
 
                 } else if (status == 'CANCELED' || status == 'FAILED' || status == 'INVALID_REQUEST' || status == 'INVALID_TOKEN') {
-                    handleError(urlParams.get("message"));
+                    handleError('<%= Encode.forHtmlContent(request.getParameter("message")) %>');
                 } else if (status == 'INVALID_USER') {
-                    handleUserError(urlParams.get("message"));
+                    handleUserError('<%= Encode.forHtmlContent(request.getParameter("message")) %>');
```

Comment:
```suggestion
                    handleUserError('<%= Encode.forJava(request.getParameter("message")) %>');
```

https://javadoc.io/static/org.owasp.encoder/encoder/1.2.3/org/owasp/encoder/Encode.html#forJava-java.lang.String-

Reply:
This is not being encoded to use in Java. It is encoded and then placed in the innerHTML[1]. Hence why Encode.forHtmlContent was used.

[1] https://github.com/Lakshan-Banneheke/identity-apps/blob/master/identity-apps-core/apps/authentication-portal/src/main/webapp/hyprlogin.jsp#L270

Reply:
Ack